app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -99,20 +99,32 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
   }
 
   private fun contactDoctorClicked(model: PatientSummaryModel): Next<PatientSummaryModel, PatientSummaryEffect> {
-    return when (model.teleconsultInfo) {
-      // TODO (SM): Check phone numbers before dispatching
-      //  LoadPatientTeleconsultationInfo to show contact doctor sheet
-      is TeleconsultInfo.Fetched -> dispatch(LoadPatientTeleconsultationInfo(
-          model.patientUuid,
-          model.patientSummaryProfile?.bpPassport,
-          model.currentFacility,
-          model.teleconsultInfo.doctorsPhoneNumbers.first()
-      ) as PatientSummaryEffect)
+    return when (val teleconsultInfo = model.teleconsultInfo) {
+      is TeleconsultInfo.Fetched -> {
+        val effect = when {
+          teleconsultInfo.doctorsPhoneNumbers.size > 1 -> {
+            OpenContactDoctorSheet(teleconsultInfo.doctorsPhoneNumbers)
```

Comment:
One more thing I forgot to add: this commit breaks the app. The reason being that the method that opens the contact patient sheet is a no-op on the UI so clicking the button when there are no doctors available will result in a button-click that does nothing.

We should squash the first three commits together so that this change goes in as a single commit.

Reply:
I have squashed the commits, but when no doctors are available the button should go to disable state. So the user won't be able to press the button.

Reply:
Sorry, I meant when more than one doctor is available.